modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml

```
@@ -240,10 +287,25 @@ Parameters:
     Default: 1
     Description: Desired number of instances in the Autoscaling Group
     Type: Number
+  EbsOptimized:
+    AllowedValues:
+      - 'true'
+      - 'false'
+    Default: 'false'
+    Description: >-
+      (Optional) Specifies whether the launch configuration is optimized for EBS I/O. This optimization provides
+      dedicated throughput to Amazon EBS and an optimized configuration stack to provide optimal EBS I/O performance
+    Type: String
   InstanceRole:
     Default: ''
     Description: (Optional) IAM instance role to apply to the instance(s)
     Type: String
+  InstanceTerminationPolicies:
+    Default: ''
+    Description: >-
+      (Optional) A policy or a list of policies that are used to select the instances to terminate.
```

Even if providing just one, it's a list with a single item...

```suggestion
      (Optional) A list of policies that are used to select the instances to terminate.
```